package.json

```diff
@@ -178,7 +178,7 @@
         "typescript": "^4.5.3",
         "yargs": "^17.2.1"
     },
-    "customElements": ".storybook/custom-elements.json",
+    "customElements": "projects/documentation/custom-elements.json",
```

Comment:
This wasn't an intentional change, but rather a side effect of development (maybe a `yarn docs:something` script?). I can revert it if the .storybook version is canonical.

Reply:
Related to https://github.com/open-wc/custom-elements-manifest/issues/121 don't worry about it here.